infra/modules/database/role_manager/check.py

```diff
@@ -47,6 +49,20 @@ def check_app_use_table(app_conn: Connection):
     db.execute(app_conn, "SELECT * FROM role_manager_test")
 
 
+def check_superuser_extensions(app_conn: Connection, superuser_extensions: dict):
+    def to_str(enabled):
+        return "enabled" if enabled else "disabled"
+
+    for extension, should_be_enabled in superuser_extensions.items():
+        print(f"-- Check that {extension} extension is {to_str(should_be_enabled)}")
+        result = db.execute(app_conn, f"SELECT * FROM pg_extension WHERE extname={literal(extension)}")
+        is_enabled = len(result) > 0
+        if (should_be_enabled and is_enabled) or (not should_be_enabled and not is_enabled):
+            print(f"---- Success, {extension} is {to_str(is_enabled)}")
+        else:
+            print(f"---- Warning, {extension} is {to_str(is_enabled)}")
```

Comment:
This could also act more strongly and raise an exception if it differed (which I expect is what the other code would do)

Reply:
if you don't want to refactor to returning success: false, the easiest for now might be to just collapse the entire if block into an assert e.g.

```
assert is_enabled == should_be_enabled
```

Reply:
I'm gonna take this approach!